.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,14 @@
+name: golangci-lint
+on:
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
```

Comment:
Any way we can keep the versions in sync with the `Makefile`? (https://github.com/projectcontour/contour/blob/main/hack/golangci-lint#L15)

Reply:
I did not quite understand what you mean. Do you mean is there any way to sync these two versions or syncing it now from v1.29 to v.1.31.0?

Reply:
I can't think of a great way to have only a single place with the version. We might just have to live with it in multiple places like we do with the Go version :(

Reply:
There's a version in the hacks/golanglint-ci script as well as now in this GH Action PR. So we need to be diligent in keeping the two in in sync. This is the benefit of just leaving the Makefile doing the work and not having a separate action.